README.md

```diff
@@ -16,42 +18,177 @@ or a given CSV in the same format with the
 [cisagov/hash-http-content](https://github.com/cisagov/hash-http-content)
 Python library. Then it will output CSVs with agency and domain level results.
 
-## Usage ##
+## Running ##
 
-### Install ###
+### Running with Docker ###
 
-Pull `cisagov/vdp-scanner` from the Docker repository:
+To run the `cisagov/vdp-scanner` image via Docker:
 
 ```console
-docker pull cisagov/vdp-scanner
+docker run cisagov/vdp-scanner:0.0.2
 ```
 
-Or build `cisagov/vdp-scanner` from source:
+### Running with Docker Compose ###
 
-```console
-git clone https://github.com/cisagov/vdp-scanner-docker.git
-cd vdp-scanner-docker
-docker-compose build
-```
+1. Create a `docker-compose.yml` file similar to the one below to use [Docker Compose](https://docs.docker.com/compose/).
 
-### Run ###
+    ```yaml
+    ---
+    version: "3.7"
 
-This Docker image needs a bind mount to get the output from the script to the
-host.
+    services:
+      vdp-scanner:
+        image: 'cisagov/vdp-scanner:0.0.2'
+        volumes:
+          - .:/task/host_mount
+    ```
 
-Using `docker run`
+1. Start the container and detach:
 
-```console
-docker run --mount type=bind,source=$(pwd),target=/task/host_mount --rm cisagov/vdp-scanner
-```
+    ```console
+    docker-compose up --detach
+    ```
+
+## Updating your container ##
+
+### Docker Compose ###
+
+1. Pull the new image from Docker hub:
```

Comment:
Use preferred capitalization:
```suggestion
1. Pull the new image from Docker Hub:
```

Reply:
That is pulled in from [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker/blame/develop/README.md#L105) so I'd like to fix it there and let [cisagov/action-lineage](https://github.com/cisagov/action-lineage) push it down.

Reply:
Good call - it's going to be handled in https://github.com/cisagov/skeleton-docker/pull/65.